From a217f5e3449eaa8c9d22797f1cd7ec960514e8b2 Mon Sep 17 00:00:00 2001
From: Sven van Haastregt <sven.vanhaastregt@arm.com>
Date: Wed, 9 Oct 2024 17:20:45 +0200
Subject: [PATCH 17/18] UnXFAIL untyped_var test after spirv-tools update
 (#2748)

With the latest release of spirv-tools (1.3.296) this test now passes.

Original commit:
https://github.com/KhronosGroup/SPIRV-LLVM-Translator/commit/a217f5e3449eaa8
---
 .../extensions/KHR/SPV_KHR_untyped_pointers/untyped_var.spvasm | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/test/extensions/KHR/SPV_KHR_untyped_pointers/untyped_var.spvasm b/test/extensions/KHR/SPV_KHR_untyped_pointers/untyped_var.spvasm
index 224f64db20b6..604842312f85 100644
--- a/test/extensions/KHR/SPV_KHR_untyped_pointers/untyped_var.spvasm
+++ b/test/extensions/KHR/SPV_KHR_untyped_pointers/untyped_var.spvasm
@@ -1,6 +1,3 @@
-; TODO: enable back once spirv-tools are updated.
-; XFAIL: *
-
 ; REQUIRES: spirv-as
 ; RUN: spirv-as --target-env spv1.6 -o %t.spv %s
 ; RUN: spirv-val %t.spv
-- 
2.39.1

